### PR TITLE
Fix stride default value None in torch.nn.functional.avg_pool

### PR DESCRIPTION
**Pull request by pyjhzwh**
The default value of stride in nn.functional.avg_pool is set as None,
https://github.com/apache/incubator-tvm/blob/5a0f39b5481a30a2eec49e27cbc17a722bd6ee6a/python/tvm/relay/frontend/pytorch.py#L461
It will cause `TVMError: Check failed: ObjectTypeChecker<TObjectRef>: :Check(ptr): Expect relay.Expr but get Array` since `inputs[2]` is empty.
Fix it by detecting None type of stride in tvm frontend pytorch.

### CODE REVIEW

File: python/tvm/relay/frontend/pytorch.py [hunk 1]
```diff
@@ -918,7 +921,7 @@ def _get_constant(node):
 
 def _get_operator_nodes(nodes):
     """ Returns torch IR nodes that need conversion to Relay """
-    ops = {}
+    ops = []
```
Comment (masahi) [inline python/tvm/relay/frontend/pytorch.py]: You should  not have this chane in this pr. Fetch and rebase  against master.

**Comment by masahi** [comment]
Please add a test case. Add your test below. 

https://github.com/apache/incubator-tvm/blob/fc7f0783940c362bf48cd46817956381196201e2/tests/python/frontend/pytorch/test_forward.py#L370

File: python/tvm/relay/frontend/pytorch.py [hunk 2]
```diff
@@ -458,7 +458,10 @@ def _impl(inputs, input_types):
         data = inputs[0]
 
         pool_size = _infer_shape(inputs[1])
-        strides = _infer_shape(inputs[2])
+        if inputs[2]:
+            strides = _infer_shape(inputs[2])
+        else:
+            strides = pool_size
```
Comment (masahi) [inline python/tvm/relay/frontend/pytorch.py]: is this correct? we should use the default strides

Comment (pyjhzwh) [reply python/tvm/relay/frontend/pytorch.py]: https://github.com/pytorch/pytorch/blob/17a5c677963dc3ecb7ff505585ed15eadaaf74ef/torch/nn/functional.py#L269-L270
According to the description of strides above, for avg_pool2d. Stride's  default value kernel_size, which should have the same size as pool_size here

Comment (masahi) [reply python/tvm/relay/frontend/pytorch.py]: ok

**Comment by masahi** [comment]
Let's wait until the commit author issue is resolved https://discuss.tvm.ai/t/github-issue-the-commit-author-is-wrong-since-today/5880/

**Comment by masahi** [comment]
It seems the github bug was resolved, I'll try merge it.

**Comment by masahi** [comment]
Looks like it is ok.

**Pull request merged**